catroid/src/main/java/org/catrobat/catroid/ui/settingsfragments/SettingsFragment.java

```diff
@@ -476,12 +476,12 @@ private static Locale getLocaleFromLanguageTag(String languageTag) {
 
 	public static void setLanguageSharedPreference(Context context, String value) {
 		SharedPreferences.Editor editor = PreferenceManager.getDefaultSharedPreferences(context).edit();
-		editor.putString(LANGUAGE_TAG_KEY, value);
-		editor.commit();
+		editor.putString(LANGUAGE_TAG_KEY, value).apply();
 	}
 
-	public static void removeLanguageSharedPreference(Context mContext) {
+	public static void resetLanguageSharedPreference(Context mContext) {
```

Comment:
I think renaming the function to resetLanguageSharedPreference is more meaningful. since this function is used only in test Cases, would be better to set the language to the defaultSystemLanguage, which is stored in a static var in CatroidApplication.